### PR TITLE
Datadog output

### PR DESCRIPTION
**Pull request by udgover**
Updates taking into account Christophe's remarks.

### CODE REVIEW

File: etc/cowrie.cfg.dist [hunk 1]
```diff
@@ -1058,6 +1058,6 @@ url = https://discord.com/api/webhooks/id/token
 enabled = false
 url = https://http-intake.logs.datadoghq.com/api/v2/logs
 api_key = abcdef1234567890fedcba0987654321
-ddsource = "cowrie"
-ddtags = "env:prod,version:2.1.0"
-service = "honeypot"
+ddsource = cowrie
+ddtags = env:dev
```
Comment (christophetd) [inline etc/cowrie.cfg.dist:1062]: (I assume you don't need quotes)

File: src/cowrie/output/datadog.py [hunk 2]
```diff
@@ -20,9 +21,11 @@
 class Output(cowrie.core.output.Output):
     def start(self):
         self.url = CowrieConfig.get("output_datadog", "url").encode("utf8")
-        self.api_key = CowrieConfig.get("output_datadog", "api_key").encode("utf8")
+        self.api_key = CowrieConfig.get("output_datadog", "api_key", fallback="").encode("utf8")
+        if len(self.api_key) == 0:
+            log.msg("Datadog output module: API key is not defined.")
```
Comment (christophetd) [inline src/cowrie/output/datadog.py:26]: you'd likely need to exit here? not only log

**Review by christophetd — APPROVED** [review]

File: src/cowrie/output/datadog.py (hunk 2 above)
Comment (udgover) [reply src/cowrie/output/datadog.py:26]: You can't.

File: etc/cowrie.cfg.dist (hunk 1 above)
Comment (udgover) [reply etc/cowrie.cfg.dist:1062]: Exactly

**Pull request merged**